### PR TITLE
Improve docker instructions in docs

### PR DESCRIPTION
**Pull request by krzysztofwolski**
Added missing steps for docker compose.

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -100,33 +102,54 @@ You will need to install Docker and
...
+   .. code-block:: bash
+
+    $ yarn
+
+
+2. Prepare static assets
+
+   .. code-block:: bash
+
+    $ npm run build-assets
```
Comment (bogdal) [inline docs/development.rst]: We should use `yarn` instead of `npm`:
```
yarn run build-assets
```

File: docs/development.rst [hunk 2]
```diff
@@ -100,33 +102,54 @@ You will need to install Docker and
...
     $ docker-compose build
 
 
-2. Prepare the database
+4. Prepare the database
 
    .. code-block:: bash
 
     $ docker-compose run web python manage.py migrate
+    $ docker-compose run web python manage.py collectstatic --noinput
```
Comment (bogdal) [inline docs/development.rst]: We don't need to use `--noinput` parameter on local env.

File: docs/development.rst [hunk 3]
```diff
@@ -100,33 +102,54 @@ You will need to install Docker and
 `docker-compose <https://docs.docker.com/compose/install/>`_ before
 performing the following steps.
 
+To build assets you will need `node <https://docs.docker.com/compose/install/>`_, `yarn <https://docs.docker.com/compose/install/>`_ and `webpack module bundler <https://webpack.github.io/>`_.
+
+.. note::
+
+   Our configuration exposes database, Redis and Elasticsearch ports. If you have problems running this docker file because of port conflicts, you can remove 'ports' section from docker-compose.yml
```
Comment (bogdal) [inline docs/development.rst]: I think `Postgresql` instead of `database` may fit better here.

Comment (patrys) [inline docs/development.rst]: PostgreSQL, always use official names.

**Comment by krzysztofwolski** [comment]
@bogdal Changed!

**Pull request merged**